src/plugins/kibana_overview/public/components/overview/overview.tsx

```diff
@@ -72,6 +72,16 @@ export const Overview: FC<Props> = ({ newsFetchResult, solutions, features }) =>
   const addDataFeatures = getFeaturesByCategory(FeatureCatalogueCategory.DATA);
   const manageDataFeatures = getFeaturesByCategory(FeatureCatalogueCategory.ADMIN);
   const devTools = findFeatureById('console');
+  const noDataConfig: KibanaPageTemplateProps['noDataConfig'] = {
```

Comment:
nit: Could import `NoDataPageProps` directly instead of accessing it by index here.

Reply:
Have a rough example of how that would look?

Reply:
`noDataConfig` uses the `NoDataPageProps` interface, which I think is already exported from `kibana_react`: 
https://github.com/elastic/kibana/blob/7fb30ba6f41ee2d1cfd264b08a4f8cde34d9f73e/src/plugins/kibana_react/public/page_template/page_template.tsx#L44-L48

So you could do
```suggestion
  const noDataConfig: NoDataPageProps = {
```

Again, just a stylistic nit, so strong feelings here :)

Reply:
Tried this out then ultimately switched back as other solutions are following the same pattern - https://github.com/elastic/kibana/pull/107709